doc/source/User_guide/pyaedt_extensions_doc/hfss3dlayout/parametrize_edb.rst

```diff
@@ -0,0 +1,34 @@
+Parametrize Layout
+==================
+
+You can parametrize stackup, materials, padstacks and traces of an existing 3D Layout design and also
+change size of voids and polygons to conduct corner analysis.
+
+You can access the extension from the icon created on the **Automation** tab using the Extension Manager.
+
+The following image shows the extension user interface:
+
+.. image:: ../../../_static/extensions/parametrize.png
```

Comment:
where is the image? I can't see it as an added file

Reply:
images have been pushed now.